### PR TITLE
port 5.4.1

### PR DESCRIPTION
**Pull request by aarongranick-okta**
### Reviewers:
@jmelberg-okta 

### Issue:

- [OKTA-375492](https://oktainc.atlassian.net/browse/OKTA-375492)

### CODE REVIEW

File: src/widget/buildRenderOptions.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  if (!renderOptions.clientId) {
+    throw new Errors.ConfigError('"clientId" is required');
+  }
+
+  if (!renderOptions.redirectUri) {
+    throw new Errors.ConfigError('"redirectUri" is required');
+  }
+
+  return renderOptions;
+}
```
Comment (jmelberg-okta) [inline src/widget/buildRenderOptions.js:23]: style-lint: Newlines at the eof are generally preferred. Surprised we don't have a lint rule for this!

File: scripts/jest.sh [hunk 2]
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-setup_service google-chrome-stable 83.0.4103.61-1
-
 source $OKTA_HOME/$REPO/scripts/setup.sh
```
Comment (jmelberg-okta) [inline scripts/jest.sh:3]: Do we intend to include this in our `.travis.yml` config as well?

**Review by jmelberg-okta — APPROVED** [review]
Changes from the Symantec VIP authenticator look correct!

We have the opportunity to enable a few more lint rules to cleanup some common `style-lint` errors, maybe we pick this up sometime next week.

**Review by haishengwu-okta — APPROVED** [review]

File: scripts/jest.sh (hunk 2 above)
Comment (aarongranick-okta) [reply scripts/jest.sh:3]: sounds like a reasonable proposition

**Pull request closed without merging**